thorvg/examples/thorvg-example/sdkconfig.defaults

```diff
@@ -0,0 +1,17 @@
+# This file was generated using idf.py save-defconfig. It can be edited manually.
+# Espressif IoT Development Framework (ESP-IDF) Project Minimal Configuration
+#
+CONFIG_IDF_TARGET="esp32p4"
+
+CONFIG_ESPTOOLPY_FLASHMODE_QIO=y
+CONFIG_ESPTOOLPY_FLASHSIZE_4MB=y
+CONFIG_PARTITION_TABLE_CUSTOM=y
+CONFIG_COMPILER_OPTIMIZATION_PERF=y
+CONFIG_COMPILER_CXX_EXCEPTIONS=y
+CONFIG_SPIRAM=y
+CONFIG_SPIRAM_SPEED_200M=y
```

Comment:
they're target specific, add them in the sdkconfig.defaults.esp32p4 and sdkconfig.defaults.esp32s3